how-to/running.rst

```diff
@@ -417,6 +419,27 @@ the bridge interface on your computer):
     sudo ufw allow in on lxdbr0
     sudo ufw route allow in on lxdbr0
 
+.. _pynacl-fix:
+
+Error building Python wheels
+----------------------------
+
+When running ``make schema`` on some machines, ``pynacl`` `fails to build <https://github.com/pyca/pynacl/issues/553>`_, leading to ``ERROR: Failed building wheel for pynacl``.
+
+If you encounter this issue, try running the following:
+
+.. code-block:: shell-session
+
+    $ sudo apt install --yes libsodium-dev
```

Comment:
I usually tend to not add the `-y` flag in these commands to make it more of a conscious decision that users install an extra package, but that's a nit